packages/core/src/dashboards/component/AOPChart/AOPChart.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useEffect, useState} from 'react';
+import {Dimensions, View} from 'react-native';
+import {Chart, BarChart, LineChart, PieChart} from '@axelor/aos-mobile-ui';
+import {
+  fetchActionView,
+  fetchChartDataset,
+  fetchTypeChart,
+  getChartParameter,
+} from '../../../api/chart-api';
+
+interface AOPChartProps {
+  actionViewName: string;
+}
+
+const AOPChart = ({actionViewName}: AOPChartProps) => {
+  const [chart, setChart] = useState({type: '', dataset: [], title: ''});
```

Comment:
Default values are used at three places, can you create a constant for that please 
```
const DEFAULT_CONFIG = {type: '', dataset: [], title: ''};
```